cocos/core/director.ts

```diff
@@ -927,6 +927,21 @@ export class Director extends EventTarget {
         }
     }
 
+    /**
+     * @en
+     * Run once main loop.
+     * @zh
+     * 运行一次主循环。
+     * @param deltaTime 此次循环与上次循环间隔的时间，单位为毫秒。默认为自然流逝的时间。
+     */
+    public tick (deltaTime?: number) {
+        this.mainLoop(
+            typeof deltaTime === 'undefined'
+                ? undefined
+                : this._lastUpdate + deltaTime,
+        );
+    }
```

Comment:
is it necessary to add a new interface for developer ?

Reply:
I suggest to change this interface to a private method, because it's not  a useful interface for developper

Reply:
I suggest to change this interface to a private method, because it's not a useful interface for developper

Reply:
I suggest to change this interface to a private method, because it's not a useful interface for developper

Reply:
It is a useful method, and we should use the new `tick` function instead of `mainLoop`.
`tick` accept dt, which means developer can easier control speed of game globally.